neuralprophet/forecaster.py

```diff
@@ -2848,17 +2852,73 @@ def conformal_predict(
         df_cal = self.predict(calibration_df)
         # get predictions for test dataframe
         df_test = self.predict(df, **kwargs)
+        df_qr = df_test.copy()
+
         # initiate Conformal instance
         c = Conformal(
             alpha=alpha,
             method=method,
             n_forecasts=self.n_forecasts,
             quantiles=self.config_train.quantiles,
         )
-        # call Conformal's predict to output test df with conformal prediction intervals
-        df_forecast = c.predict(df=df_test, df_cal=df_cal)
+
+        if not show_qr:
+            # call Conformal's predict to output test df with conformal prediction intervals
+            df_forecast = c.predict(df=df_test, df_cal=df_cal)
+            # plot one-sided prediction interval width with q
+        else:
+            df_forecast = c.predict(df=df_test, df_cal=df_cal)
+
+            # get quantile regression intervals
+            df_quantiles = df_qr.columns[df_qr.columns.str.contains("%")]
+            df_add = df_qr[df_quantiles]
+            df_add.columns = [f"qr_{col}" for col in df_add.columns]
+            df_forecast = pd.concat([df_forecast, df_add], axis=1, ignore_index=False)
```

Comment:
For naive CP without `show_qr` (basic users), the columns are `['ds', 'y', 'yhat1', 'yhat1 5.0%', 'yhat1 95.0%', 'trend',
       'season_weekly', 'season_daily']`. Where `yhat1 5.0%`, and `yhat1 95.0%` are the CP PI that replaced the QR ones (if existed). That's fine, as basic users only care about the PI regardless of the technique.

However, for naive CP with `show_qr` (advanced users), the columns are now `['ds', 'y', 'yhat1', 'yhat1 5.0%', 'yhat1 95.0%', 'trend',
       'season_weekly', 'season_daily', 'qr_yhat1 5.0%', 'qr_yhat1 95.0%']`. That is very confusing for me because it took a while for me to register that `yhat1 5.0%` and `yhat1 95.0%` refers to the CP PI. Therefore, I suggest that for advanced users, you use my existing naming convention. 

- `yhat1 <quantile>` for QR PI
- `yhat1 +- qhat1` for naive CP PI
- `yhat1 <quantile> +- qhat1` for CQR PI

I have this naming convention for a reason; it easily distinguishes the different PIs from each other. And it especially distinguishes naive CP and CQR as their qhats are used on the mean and the QR quantiles respectively. Therefore, their qhats are not used in the same way hence the difference naming columns.

Because of this, I also suggest the change the naming of param `show_qr` to something else. Like show all PIs and show complete uncertainty etc.

Reply:
I implemented all demanded changes, but for this one, I want to outline the consequences this one would have.
If the naming goes back to `yhat <quantile>` (qr), `yhat1 +- qhat1` (naive CP) and `yhat1 <quantile> +- qhat1` (CQR PI), then:

- An user that used `show_all_PI = True` is not able to use the normal plotting method anymore as the normal plotting method only plots `yhat <quantile>` uncertainty intervals
- For someone starting with CQR, I think it's very confusing, but with the proposed naming you can easily see which one is the most current method and which one stems from original qr. 

But if you want those changes, although it would limit the use of the normal plotting method (only `conformal_plot` would be possible then), I can implement those.

Reply:
done